package.json

```diff
@@ -99,6 +99,7 @@
     "sprintf-js": "~1.1.1",
     "table-resolver": "^4.1.1",
     "ui-select": "0.19.8",
+    "webpack-stats-plugin": "^0.3.2",
```

Comment:
can you move this one to devDependencies please?
(to distinguish build-time and run-time deps)

Reply:
This is a build-time dep as I need to get the stats from a production build.  (unless dev deps are used at build time?)

Reply:
> dev deps are used at build time

exactly that :) so that runtime deps are the default ones

Reply:
Updated... I didn't realize even webpack was a dev dep as I assumed we'd install only prod deps in a prod build.